scripts/code.angularjs.org-firebase/firebase.json

```diff
@@ -0,0 +1,21 @@
+{
+  "hosting": {
+    "public": "public",
+    "redirects": [
+      {
+        "source": "/:version/docs",
+        "destination": "/:version/docs/index.html",
+        "type": 301
```

Comment:
For some reason, redirect to `/:version/docs/` results in an infinite redirect loop in the browser 🤔